lib/fsm/door_fsm.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef PET_PASSAGE_FSM_DOOR_FSM_H_
```

Comment:
For your #ifndef and #define there is an inconsistency. In the files under lib/hardware you use 
 `#ifndef PET_PASSAGE_LIB_MAGNETIC_SENSOR_H_ `
Noting LIB here but not hardware subfile, though in these files you do not cite LIB but FSM. Review the style guide for the right form for consistency?

Reply:
Based on the example it provides of `foo/src/bar/baz.h` becoming `FOO_BAR_BAZ_H_` I believe the presence of LIB was a mistake in `PET_PASSAGE_LIB_MAGNETIC_SENSOR_H_`. However, this should be addressed in a seperate PR.